docs/tutorial/part-one/index.md

```diff
@@ -226,35 +228,35 @@ export default () => (
 
 ![Passing data to header](07-pass-data-header.png)
 
-You should now see your “About Gatsby” header text again!
+अब आपको अपना “About Gatsby”  हेडर टेक्स्ट फिर से दिखना चाहिए!
 
-### What are “props”?
+### “प्रॉप्स” क्या है?
 
-Earlier you defined React components as reusable pieces of code describing a UI. To make these reusable pieces dynamic you need to be able to supply them with different data. You do that with input called “props". Props are (appropriately enough) properties supplied to React components.
+इससे पहले आपने रिक्ट कौम्पोनॅन्ट को कोड के रियूज़बल पीसस के रूप में परिभाषित करते हुए UI के रूप मे वर्णन किया था। इन रियूज़बल पीसस को गतिशील बनाने के लिए आपको अलग-अलग डेटा के साथ सप्लाई करना होगा। आप ऐसा करते हैं "प्रॉप्स"  इनपुट के साथ। प्रॉप्स(उचित रूप से पर्याप्त) दरअसल प्रॉपर्टीस है जिसे रिक्ट कौम्पोनॅन्ट को सप्लाई करते है।
 
-In `about.js` you passed a `headerText` prop with the value of `"About Gatsby"` to the imported `Header` sub-component:
+`about.js` में आपने `headertext` प्रॉप्स में `"About Gatsby"` को  वॅल्यू के रूप मे `Header` सब-कौम्पोनॅन्ट मे भेजा था जो की आपने इंपोर्टे किया था:
 
 ```jsx:title=src/pages/about.js
 <Header headerText="About Gatsby" />
 ```
 
-Over in `header.js`, the header component expects to receive the `headerText` prop (because you’ve written it to expect that). So you can access it like so:
+`header.js` में, हेडर कौम्पोनॅन्ट `headerText` प्रॉप को प्राप्त करने की उम्मीद करता है (क्योंकि आपने यह उम्मीद करने के लिए इसे लिखा है)। तो आप इसे इस तरह एक्सेस कर सकते हैं:
 
 ```jsx:title=src/components/header.js
 <h1>{props.headerText}</h1>
 ```
 
-> 💡 In JSX, you can embed any JavaScript expression by wrapping it with `{}`. This is how you can access the `headerText` property (or “prop!”) from the “props” object.
+> 💡 JSX में, आप किसी भी Javascript एक्सप्रेशन को `{}` के साथ रैप करके एम्बेड कर सकते हैं। इस प्रकार आप “प्रॉप्स” ऑब्जेक्ट से `headerText` प्रॉपर्टी (या “प्रोप!”) को एक्सेस कर सकते हैं।
 
-If you had passed another prop to your `<Header />` component, like so...
+यदि आपने अपने `<Header />` कौम्पोनॅन्ट मे एक और “प्रोप” उपयोग किया है, तो ...
```

Comment:
`“प्रोप”` => `“prop”`

Reply:
> `“प्रोप”` => `“prop”`

Actually this should be

`“prop”` => `prop`

without quotes

Reply:
i think, this is done.
![image](https://user-images.githubusercontent.com/27020381/70386097-0dd71680-19bb-11ea-82ad-ec40e6b0f525.png)

Reply:
or do you want to remove double quote for ```prop/props`` everywhere? let me know

Reply:
> or do you want to remove double quote for ```prop/props`` everywhere? let me know

just this one, to keep it same as english